internal/action/sync.go

```diff
@@ -69,13 +71,20 @@ func (s *Action) autoSync(ctx context.Context) error {
 
 	ls := s.rem.LastSeen("autosync")
 	debug.Log("autosync - last seen: %s", ls)
-	syncInterval := autosyncIntervalDays
+	syncInterval := autosyncInterval
 
-	if s.cfg.IsSet("autosync.interval") {
-		syncInterval = s.cfg.GetInt("autosync.interval")
+	if intervalStr := s.cfg.Get("autosync.interval"); intervalStr != "" {
+		if _, err := strconv.Atoi(intervalStr); err == nil {
+			intervalStr += "d"
+		}
+		duration, err := str2duration.Str2Duration(intervalStr)
+		if err != nil {
+			return exit.Error(exit.Usage, err, "failed to parse autosync.interval %q: %q", intervalStr, err)
```

Comment:
This causes no error, because it is not cared for at https://github.com/gopasspw/gopass/blob/master/internal/action/init.go#L44  -- what to do about it?

Reply:
Maybe output a warning using `out.Warn` in case of an error?

Reply:
But this implies a syntax error with the autosync.interval the config file. And what autosync.interval should we then continue running with?

Reply:
As an invalid autosync interval shouldn't be a fatal error we should continue running with a sensible default (e.g. 3 days).

Reply:
I've pushed a commit that falls back to the default. Other review comments remain...